settings.gradle.kts

```diff
@@ -25,25 +25,32 @@ dependencyResolutionManagement {
     }
 }
 rootProject.name = "DroidKaigi2022"
-include(
-    ":app-android",
-    ":core-zipline",
-    ":appioscombined",
-    ":feature-sessions",
-    ":feature-contributors",
-    ":feature-about",
-    ":feature-map",
-    ":feature-announcement",
-    ":feature-setting",
-    ":feature-staff",
-    ":feature-sponsors",
-    ":core-ui",
-    ":core-designsystem",
-    ":core-data",
-    ":core-testing",
-    ":core-model",
-    ":preview-screenshots"
+val modules = listOf(
+    "app-android",
+    "core-zipline",
+    "feature-sessions",
+    "feature-contributors",
+    "feature-about",
+    "feature-map",
+    "feature-announcement",
+    "feature-setting",
+    "feature-staff",
+    "feature-sponsors",
+    "core-ui",
+    "core-designsystem",
+    "core-data",
+    "core-testing",
+    "core-model",
+    "preview-screenshots"
 )
 
+modules.forEach { filePath ->
+    val (type, name) = filePath.split("-")
+    val newProjetName = ":$type:$name"
```

Comment:
Using `:` might make the iOS bundle cringed ... iOS bundle use projectname so it would turn into something like `DroidKaigi2022.core:model.bundle` ... in my opinion using `.` is probably the best here.

Reply:
Thanks. I'm trying it 🙏 
https://github.com/DroidKaigi/conference-app-2022/pull/585

Reply:
For now, I decided to revert this.

Reply:
I managed to fix the issue through this PR. But your pr is still failing. 
https://github.com/DroidKaigi/conference-app-2022/pull/584/files
If we do this, It seems like our project can't use the project accessor.
https://github.com/DroidKaigi/conference-app-2022/pull/585